packages/host/app/components/operator-mode/definition-container.gts

```diff
@@ -0,0 +1,184 @@
+import Label from '@cardstack/boxel-ui/components/label';
+import Component from '@glimmer/component';
+import { on } from '@ember/modifier';
+import { svgJar } from '@cardstack/boxel-ui/helpers/svg-jar';
+import { Button } from '@cardstack/boxel-ui';
+import { eq, and } from '@cardstack/boxel-ui/helpers/truth-helpers';
+import { assertNever } from '@cardstack/host/utils/assert-never';
+// boxel- specific
+import { type RealmInfo } from '@cardstack/runtime-common';
+
+export enum DefinitionVariant {
+  Instance = 'instance',
+  Module = 'module',
+}
+
+interface Signature {
+  Element: HTMLElement;
+  Args: {
+    name: string | undefined;
+    fileExtension: string;
+    realmInfo: RealmInfo | null;
+    realmIconURL: string | null | undefined;
+    variant: DefinitionVariant;
+    isActive: boolean;
+    infoText?: string;
+    onCreate?: () => void;
+    onInherit?: () => void;
+    onDuplicate?: () => void;
+  };
+}
+
+export default class DefinitionContainer extends Component<Signature> {
+  get title(): string {
+    switch (this.args.variant) {
+      case DefinitionVariant.Module:
+        return 'CARD DEFINITION';
```

Comment:
Since the intent of the uppercase text here is only for presentational purposes it's better to deal with that in the template - consider using lower case here and `text-transform: uppercase;`in the css for that element